### PR TITLE
Vertex SDK AutoML Text Sentiment Analysis

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Storage bucket not found issues.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb [hunk 1]
```diff
@@ -3,6 +3,7 @@
     {
```
Comment (andrewferlitsch) [inline notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb:3]: <p>Heading needs to be sentence cased</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb:3]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb:3]: <p>Install the packages required for executing this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (udaypunna) [reply notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb:3]: <p>Updated.</p>

Comment (udaypunna) [reply notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb:3]: <p>Updated.</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**